game-configs.json

```diff
@@ -87,7 +87,9 @@
     "playWaitTimeBotSoft":10000,
     "replaySave":1,
     "playReplaySave":0,
-    "storeBotTimes":true
+    "storeBotTimes":true,
+    "waitTimeStartGame":3000,
```

Comment:
```suggestion
    "waitTimeGameStart":3000,
```

Reply:
Ah I wish you commented earlier and I read it earlier because we already deployed the game engine to use these names.

Your names are definitely better though, maybe in a future push we could do it.